docs/community/http-client-integrations.md

```diff
@@ -0,0 +1,200 @@
+
+Here we have collected some of the most commonly used HTTP client libraries and how to integrate them with Polly.
+
+## Setting the stage
+
+In the examples below we will register HTTP clients into a Dependency Injection container.
+
+Each time the same resilience strategy will be used to keep the samples focused on the HTTP client library integration.
+
+<!-- snippet: http-client-integrations-handle-transient-errors -->
+```cs
+private static ValueTask<bool> HandleTransientHttpError(Outcome<HttpResponseMessage> outcome)
+=> outcome switch
+{
+    { Exception: HttpRequestException } => PredicateResult.True(),
+    { Result.StatusCode: HttpStatusCode.RequestTimeout } => PredicateResult.True(),
+    { Result.StatusCode: >= HttpStatusCode.InternalServerError } => PredicateResult.True(),
+    _ => PredicateResult.False()
+};
+
+private static RetryStrategyOptions<HttpResponseMessage> GetRetryOptions()
+=> new()
+{
+    ShouldHandle = args => HandleTransientHttpError(args.Outcome),
+    MaxRetryAttempts = 3,
+    BackoffType = DelayBackoffType.Exponential,
+    Delay = TimeSpan.FromSeconds(2)
+};
+```
+<!-- endSnippet -->
+
+Here we create a strategy which will retry the HTTP request if the status code is either 408 or greater than 500 or an `HttpRequestException` was thrown.
+
+The `HandleTransientHttpError` is a V8 port of the [`HttpPolicyExtensions.HandleTransientHttpError`](https://github.com/App-vNext/Polly.Extensions.Http/tree/master).
+
+## HttpClient based
+
+We use the [`AddResilienceHandler`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.dependencyinjection.resiliencehttpclientbuilderextensions.addresiliencehandler) method to register our resilience strategy on the built-in [`HttpClient`](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient).
+
+<!-- snippet: http-client-integrations-httpclient -->
+```cs
+ServiceCollection services = new();
+
+// Register a named HttpClient and decorate with a resilience pipeline
+services.AddHttpClient(string.Empty)
+        .ConfigureHttpClient(client => client.BaseAddress = DownstreamUri)
+        .AddResilienceHandler("httpclient_based_pipeline",
+            builder => builder.AddRetry(GetRetryOptions()));
+
+var provider = services.BuildServiceProvider();
+
+// Resolve the named HttpClient
+var httpClientFactory = provider.GetRequiredService<IHttpClientFactory>();
+var httpClient = httpClientFactory.CreateClient();
+
+// Use the HttpClient by making a request
+var response = await httpClient.GetAsync(new Uri("/408"));
+```
+<!-- endSnippet -->
+
+> [!NOTE]
+> The following packages are required to the above example:
+>
+> - [Microsoft.Extensions.DependencyInjection](https://www.nuget.org/packages/microsoft.extensions.dependencyinjection): Required for the dependency injection structures
+> - [Microsoft.Extensions.Http](https://www.nuget.org/packages/Microsoft.Extensions.Http/): Required for the `AddHttpClient` extension
```

Comment:
This is a transient dependency of Microsoft.Extensions.Http.Resilience, so we can drop the need to document this.

```suggestion
```